__tests__/api.test.js

```diff
@@ -0,0 +1,42 @@
+const app = require('../app.js');
+const request = require('supertest');
+
+const seed = require('../db/seeds/seed.js')
+const testData = require('../db/data/test-data')
+const connection = require('../db/connection.js')
+
+beforeEach(() => seed(testData))
+afterAll(() => connection.end());
+
+describe.only("GET /api/topics", () => {
+  test("200: respond with an okay message", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        expect(body).toMatchObject(testData.topicData)
+      })
+  })
+  test("200: Check that all of the columns match their respective names", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({body}) => {
+        expect(body[0]).toHaveProperty("slug", expect.any(String))
+        expect(body[0]).toHaveProperty("description", expect.any(String))
+      })
```

Comment:
As you know the length of the array is 3 would this be a more robust test if you checked each of them to ensure they all have the correct keys 🤔